### PR TITLE
Name Entity Recognition function added

### LINKED ISSUE #256: Named Entity Recognition Query
Add a Named Entity Recognition query to NLP queries. Stick to TF/Keras.

### PR DESCRIPTION
**Pull request by Piyush1416**
---
name: b) New feature: Name Entity Recognition
about: Select this if you're adding a new feature to libra.
labels: ner
---

This pull request closes #256  .

**- What I did**
> I added a new functionality to detect name entities from input dataframe

**- How I did it**
> I used the huggingface library which provides multi-lingual BERT based models for entity recognition.

**- How to verify it**
> test cases added in tests.

Inorder to use it, you can run below commands:

newClient = client('location of csv file')
ouput_dictionary_of_ners = newClient.get_ner()
<even self.model['ner'] contains the output along with pretrained model and tokenizer>

- [ ] I updated the docs.
> can you please, let me know what needs to be done here?

This pull request adds a new feature to libra. @Palashio, could you please take a look at it?

### CODE REVIEW

**Comment by Piyush1416** [comment]
it failed due to wrong dataset in test, I have updated the test file now
